.github/workflows/regression.yml

```diff
@@ -52,12 +52,12 @@ jobs:
           key: ${{ matrix.config_set }}-${{ github.sha }}
           restore-keys: ${{ matrix.config_set }}
       - name: Pre Cleanup
-        uses: docker://ghcr.io/su2code/su2/build-su2:230225-2136
```

Comment:
Update the release_management action as well to use the new images please

Reply:
Updated in c6931d7714f2b51dc699ce4bb0b53ef83e8674c7